packages/nextjs/components/NetworksDropdown.tsx

```diff
@@ -78,18 +89,22 @@ export const NetworksDropdown = ({ onChange }: { onChange: (options: any) => any
       onChange={onChange}
       components={{ Option: IconOption }}
       isSearchable={!isMobile}
-      className="max-w-xs bg-white relative text-sm w-44"
+      className="max-w-xs relative text-sm w-36"
       theme={theme => ({
         ...theme,
         colors: {
           ...theme.colors,
-          primary25: "#efeaff",
-          primary50: "#c1aeff",
-          primary: "#551d98",
+          primary25: isDarkMode ? "#401574" : "#efeaff",
+          primary50: isDarkMode ? "#551d98" : "#c1aeff",
+          primary: isDarkMode ? "#BA8DE8" : "#551d98",
+          neutral0: isDarkMode ? "#000000" : theme.colors.neutral0,
```

Comment:
Black background feels a bit strange (nitpick). In design 

I think it was `#130C25`, but would be cool if we could have a border on the options when it's open, so it does not mix with the background.

This is how it would look with `w-44` and `#130C25` background (with no border):

![image](https://github.com/BuidlGuidl/abi.ninja/assets/55535804/f98386ca-47cc-4845-ab70-54420bd2d433)

Reply:
Ohh yeah, I think this looks nice ! Maybe a very thin border ?

Reply:
Added at https://github.com/BuidlGuidl/abi.ninja/pull/93/commits/a29470692e4f22389b196afc4dbd59eded06e6f4

Reply:
Love it now!